### PR TITLE
Amendments to RPM distribution to support systemd unit file.

### PR DESCRIPTION
**Pull request by JohanSaaw**
Some amendments to enable the use of a systemd unit file for the RPM distribution.

 - The unit file itself squeezeboxserver.service
- A new version of the /etc/sysconfig/squeezeboxserver (squeezeboxserver.config)
- A new file README.systemd with some explanations about the migration from SYSV to systemd
- An amended squeezeboxserver.spec
- An amended buildme.pl (just added the two new files to be copied to the RPM source directory).

 In addition to the support of systemd, I also fixed the issues with the missing symbolic link to

- /usr/lib/perl5/site_perl/Slim for SUSE distributions
- /usr/lib64/perl5/vendor_perl/Slim for RedHat based x86_64 distributions

### CODE REVIEW

**Comment by michaelherger** [comment]
Please change your pull request to be merged to 8.2, rather than 8.1. Right now merging your PR would update 8.1 to the 8.2 content, incl. version number strings. You can modify the target branch by clicking the Edit button right of the title.

**Comment by JohanSaaw** [comment]
Hi,

OK, done. Even managed to change the title.
;-)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+- The unit file itself squeezeboxserver.service
+- A new version of the /etc/sysconfig/squeezeboxserver (squeezeboxserver.config)
+- A new file README.systemd with some explanations about the migration from SYSV to systemd
+- An amended squeezeboxserver.spec
+- An amended buildme.pl (just added the two new files to be copied to the RPM source directory).
+
+In addition to the support of systemd, I also fixed the issues with the missing symbolic link to
+
+- /usr/lib/perl5/site_perl/Slim for SUSE distributions
+- /usr/lib64/perl5/vendor_perl/Slim for RedHat based x86_64 distributions
```
Comment (michaelherger) [inline README.md]: I'd say this should be the commit message, and not a readme as part of the repository.

**Review by michaelherger — CHANGES_REQUESTED** [review]
Thanks! Looking much better with only a hand full of files changed 😀.

Please
* remove the README.md
* use the file's content as the commit message (and later on as the description for the pull request)
* push again

**Comment by mherger** [comment]
Let's give this a try - thanks a lot!

**Comment by JohanSaaw** [comment]
> 
> 
> Let's give this a try - thanks a lot!

OK, that was quick. I was still pondering on how to add that text as a commit comment.

**Pull request merged**